math_series/series.py

```diff
@@ -20,4 +20,20 @@ def lucas(n):
     elif n == 1:
         return 1
     else:
-        return(lucas(n-1)+ lucas(n-2))
\ No newline at end of file
+        return(lucas(n-1)+ lucas(n-2))
+
+def sum_series(n, x=0, y=1):
+    '''
+    Sum Series takes three arguments
+    First argument n is the position in the series you'd like to know.
+    Second argument is x which represents first number in the series.
+    Third argument is y which represents the second number in the series.
+    Then it returns the number at the position n if each number was added to the previous number given two starting numbers
+    If no x or y is givin it will return the fibonacci sequence.
+    '''
```

Comment:
A `\n` character and the end or beginning of each line here will ensure that your docstring follows the same multi-line formatting when moused over in an editor.

Normal line breaks are treated as a single space otherwise.